module/support/optimisation/WalkOptimiser/src/WalkOptimiser.cpp

```diff
@@ -19,52 +19,55 @@
 
 #include "WalkOptimiser.h"
 
-#include "message/input/ServoID.h"
+#include <fmt/format.h>
+
+#include "message/support/SaveConfiguration.h"
+#include "utility/input/ServoID.h"
 #include "utility/math/angle.h"
-#include "utility/math/optimisation/PGAoptimiser.h"
-#include "utility/support/yaml_armadillo.h"
+// #include "utility/math/optimisation/PGAoptimiser.h"
```

Comment:
Commented code?

Reply:
Intentional as an example of how to use this module since you purged all of our optimisers in the thanos finger snap.

Reply:
Probably leave a comment saying that then.